supabase/migrations/29_round_free_trial.sql

```diff
@@ -0,0 +1,256 @@
+begin;
+
+-- Compute a JSONB array of line-items detailing usage under a tenant's effective tiers.
+create or replace function tier_line_items(
+  -- Ammount of usage we're breaking out.
+  amount numeric,
```

Comment:
optional: 🤔 looking at this, perhaps it would be slightly cleaner to have the `amount` parameter be an integer and then call `ceil()` when passing the argument from the `daily_line_items` CTE below? I'm good either way.

Reply:
Yep, done